rpi5.yaml

```diff
@@ -342,3 +346,17 @@ parameters:
                 type: raw_image
                 size: 8 GiB
                 image_path: "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/%{XT_DOMD_IMAGE}-%{MACHINE}.rootfs.ext4"
+
+  DOMD_WIFI:
+    desc: "Allow wifin in domd"
```

Comment:
This puzzles me. As I see we have DOMD_ROOT and DOMD_WIFI configurations. This looks like completely different features. For example for me as user I see that I can do DOMD with rootfs and DOMD which doesn't have a rootfs but get's rootfs via WIFI. My understanding is that I (as user) can call moulin --ENABLE_WIFI and I get the same rootfs but with wifi enabled. That's why I was asking about the seaprate dtbo in meta-xt-rpi5

Reply:
Question is: do we want WIFI as an optional feature? I believe it is better to have WiFi always on in DomD. 

But maybe I am missing something.

Reply:
That bothers me either. I've left a comment in meta-xt-rpi5 https://github.com/xen-troops/meta-xt-rpi5/pull/12#discussion_r1638032629

Reply:
Option renamed to ENABLE_WIFI

Reply:
Have I make turn on wifi by default?

Reply:
Sorry I don't understand you last phrase

Reply:
May be wifi should be turned on by default?

Reply:
I think that it should be on by default.